defaults/main.yml

```diff
@@ -18,6 +18,10 @@ wimpy_application_pre_commands: []
 wimpy_application_post_commands: []
 wimpy_application_additional_units: []
 
+# CloudWatch logs
+wimpy_aws_cloudwatch_enabled: True
```

Comment:
The thing is that when something fails, it's hard to debug. This way, you can see the logs on CloudWatch to see what happened. The alternative is to ssh into the node and get the logs from systemd.

What do you guys think @aramirez-es @ismFerDev ? Should this be enabled by default or not?